test-utils/src/main/java/apoc/util/Neo4jContainerExtension.java

```diff
@@ -45,6 +45,9 @@ public Neo4jContainerExtension(String dockerImage) {
         // http on 4.0 seems to deliver a 404 first
         setDockerImageName(dockerImage);
 
+        addFixedExposedPort(7474, 7474);
+        addFixedExposedPort(7687, 7687);
+
```

Comment:
I'm not sure this is right sorry. Normally in a CI you don't want to be spanning a service in a fixed port cause a machine could have something already running there or it means you cannot parallelize several test suites. If you don't map the ports docker will choose one randomly for you and you can see which one was it by doing:
```
neo4jContainer.getMappedPort(7687)
```